### PR TITLE
first approach to book index

### PR DESCRIPTION
**Pull request by osvaldoFernandez92**
Summary
Book index with pagination.

### CODE REVIEW

**Comment by osvaldoFernandez92** [comment]
@sbalay

File: src/app/components/book/bookController.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  	console.log(err);
+    alert('listin books error');
+  };
+
+  this.loadPage = function() {
+    bookService.availableBooks(successCb, errorCb, this.main.page, this.main.take);
+  };
+  
+  this.nextPage = function() {
+    if (this.main.page < 1) {
```
Comment (sbalay) [inline src/app/components/book/bookController.js:24]: Why do you assume that there will only be two pages?

File: src/app/components/book/bookController.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
+angular.module('app').controller('BookController', ['bookService',
+function(bookService) {
+
+  this.main = {
```
Comment (sbalay) [inline src/app/components/book/bookController.js:4]: Don't attach local variables to the controller scope. Declare them using let or const accordingly. Only variables or method that will be used in the view are declared in the controller scope.

File: src/app/components/book/bookController.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  };
+  
+  this.previousPage = function() {
+    if (this.main.page > 0) {
+      this.main.page--;
+      this.loadPage();
+    }
+  };
+
+  this.init = function () {
```
Comment (sbalay) [inline src/app/components/book/bookController.js:37]: Is this being invoked somewhere?

File: src/app/services/bookService.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+angular.module('app').factory('bookService', [
+  '$http', '$state', 'Restangular',
+  function ($http, $state, Restangular) {
+    return {
+       availableBooks: (success, error, page, take) => {
+         return Restangular.one('classes/Book').get({skip: page, limit: take}).then(success, error);
```
Comment (sbalay) [inline src/app/services/bookService.js:6]: Don't use slashes when declaring elements or collections in restangular

**Comment by sbalay** [comment]
@osvaldoFernandez92 comments

**Comment by sbalay** [comment]
Can you close this pull request?

**Pull request closed without merging**